### PR TITLE
Dependabot automerge

### PR DESCRIPTION
**Pull request by kyledurand**
Reduce toil. Will automerge patch and minor updates

### CODE REVIEW

**Review by alex-page — APPROVED** [review]
Looks good. As a follow up it would be nice to auto label skip changelog and merge when CI is green.

**Comment by kyledurand** [comment]
https://cli.github.com/manual/gh_pr_merge
> --auto
Automatically merge only after necessary requirements are met

File: .github/workflows/dependabot-auto-merge.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    runs-on: shopify-ubuntu-latest
+    if: ${{ github.event.pull_request.user.login == 'dependabot[bot]' }}
+    steps:
+      - name: Dependabot metadata
+        id: dependabot-metadata
+        uses: dependabot/fetch-metadata@v2
+        with:
+          github-token: '${{ secrets.GITHUB_TOKEN }}'
+
+      - name: Approve and /shipit Dependabot PRs for patch versions
```
Comment (francisbeaudoin) [inline .github/workflows/dependabot-auto-merge.yml]: Nitpick: Would suggest renaming as it implies that it is only merging patch

Comment (kyledurand) [reply .github/workflows/dependabot-auto-merge.yml]: Oh yeah I changed my mind on that. Thanks for catching

**Review by francisbeaudoin — APPROVED** [review]

**Review by sam-b-rose — APPROVED** [review]

**Pull request merged**